<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1269</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="leftMargin" >
     <number>9</number>
    </property>
    <property name="topMargin" >
     <number>9</number>
    </property>
    <property name="rightMargin" >
     <number>9</number>
    </property>
    <property name="bottomMargin" >
     <number>9</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1269</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <addaction name="actionDesign_View" />
    <addaction name="actionTrain_View" />
    <addaction name="actionTest_View" />
    <addaction name="separator" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_Qt" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionOpen_Network" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuView" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>View Bar</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Network" />
   <addaction name="separator" />
   <addaction name="actionDesign_View" />
   <addaction name="actionTrain_View" />
   <addaction name="actionTest_View" />
   <addaction name="separator" />
   <addaction name="actionZoom_In" />
   <addaction name="actionZoom_Out" />
  </widget>
  <action name="actionDesign_View" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons_32x32/images/32x32/package_utilities.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Design View</string>
   </property>
  </action>
  <action name="actionTrain_View" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons_32x32/images/32x32/package_edutainment.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Train View</string>
   </property>
  </action>
  <action name="actionStart_Train" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons_32x32/images/32x32/noatun.png</iconset>
   </property>
   <property name="text" >
    <string>Train</string>
   </property>
  </action>
  <action name="actionStop_Train" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons_32x32/images/32x32/krec.png</iconset>
   </property>
   <property name="text" >
    <string>Stop</string>
   </property>
  </action>
  <action name="actionZoom_In" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons_32x32/images/32x32/viewmag+.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons_32x32/images/32x32/viewmag-.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionOpen_Network" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons_32x32/images/32x32/fileopen.png</iconset>
   </property>
   <property name="text" >
    <string>Open Network...</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons_16x16/images/16x16/exit.png</iconset>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_Qt" >
   <property name="text" >
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionTest_View" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons_32x32/images/32x32/apply.png</iconset>
   </property>
   <property name="text" >
    <string>Test &amp;View</string>
   </property>
  </action>
  <action name="actionTest_Network" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons_32x32/images/32x32/noatun.png</iconset>
   </property>
   <property name="text" >
    <string>Test Network</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc" />
 </resources>
 <connections/>
</ui>
